canonical-web-and-design/maas-ui
# fix(legacy): Add focal to KVMDeployOSBlacklist.

## Done

- Adds focal to the list of blacklisted releases for KVM hosts.
- Updates legacy's eslint configuration to use es2020, as we build with babel/preset-env.

## QA

### MAAS deployment

To run this branch you will need access to one of the following MAAS deployments:

- [Karura](/HACKING.md#karura)
- [Bolla](/HACKING.md#bolla)
- [A development MAAS](/HACKING.md#development-deployment)
- [A local snap MAAS](/HACKING.md#snap-deployment) (this will not usually have machines)

### Running the branch

You can run this branch by:

- Serving with [dotrun](/HACKING.md#maas-ui-development-setup)
- [Building in a development MAAS](/HACKING.md#running-maas-ui-from-a-development-maas)

### QA steps

- Deploy a machine from machine details.
- Select 20.04
- Ensure "Register as MAAS KVM host" checkbox is disabled and "Ubuntu 18.04 is required to create a KVM host." error displays.

## Fixes

Fixes: #1466 

## Launchpad issue

Related Launchpad maas issue in the form `lp#number`.

## Backports

In general, please propose fixes against *master* rather than release branches (e.g. 2.7), unless the fix is only applicable for that specific release. Please apply backport labels to the PR (e.g. "Backport 2.7") for the appropriate releases to target.

Only bug and security fixes should be backported, new features should only land in master.
